### PR TITLE
Nerfs phytosians

### PR DESCRIPTION
**Pull request by Time-Green**
I think I might've found the right nerf for them:

## Nerf:

- Phytosiansnow need to eat.(Plants need more then just light)

- They loose nutrition on healing.

## Buff?:

- They take less damage from starving, since they will be dealing with it alot. 

:cl:
tweak: Phytosians realised nutriment doesn't come from lights, so they now also eat in order to stay fit.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 1]
```diff
@@ -349,72 +349,80 @@ datum/species/lizard/before_equip_job(datum/job/J, mob/living/carbon/human/H)
...
 	if(H.nutrition < NUTRITION_LEVEL_STARVING + 50)
 		if (H.stat != UNCONSCIOUS && H.stat != DEAD)
 			if (prob(5))
-				H << "<span class='userdanger'>Your internal stores of light are depleted. Find a source to replenish your nourishment at once!</span>"
-			H.take_overall_damage(2,0)
+				H << "<span class='userdanger'>Your internal stores of nutriment are depleted. Find a source to replenish your nourishment at once!</span>"
+			H.take_overall_damage(1,0)
+
+/datum/species/plant/proc/regen(var/light, mob/living/carbon/human/H)
+	world << "1"
```
Comment (Super3222) [inline code/modules/mob/living/carbon/human/species_types.dm]: You forgot to remove some of your testing checks.

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 2]
```diff
@@ -349,72 +349,80 @@ datum/species/lizard/before_equip_job(datum/job/J, mob/living/carbon/human/H)
...
+		H.adjustOxyLoss(-1.5 * light) //example: -1.5 * -2 = 3. So do 3 oxy damage.
+		world << "2"
+	else if (light > 0)
+		world << "6"
+		if(H.nutrition > 100) //If the person is hungry, they wont regen because that'd be more efficient with nutriment use.
+			H.adjustToxLoss(-0.5 * light)
+			H.adjustOxyLoss(-0.5 * light)
+			H.heal_overall_damage(1 * light, 1 * light) //Just heal them in-case something breaks
+			world << "3"
+			if((!H.health == H.maxHealth) && (prob(50))) //We use nutrition to heal, but not when we are already at full health
```
Comment (Super3222) [inline code/modules/mob/living/carbon/human/species_types.dm]: prob 50 to lose nutrition... prob 50 almost every second. they're still losing nutrition fast.

**Comment by Super3222** [comment]
Overall I don't like it. Phytosians are plant based species. Not needing to eat, and gaining nutrition from light is one of their gimmicks. Making them eat a lot is redundant from it's counterpart because phytosians will ALWAYS BE HEALING which is a problem. 

I don't see any reason for this change either. There's nothing wrong with phytosians ATM, and we have a species rework that still needs to be merged.

**Comment by matskuman5** [comment]
Agreeing with Super here. Instaed of making them eat, how about increasing their burn vulnerability a lot? So they can be killed in 2-3 laser shots?

File: code/modules/mob/living/carbon/human/species_types.dm (hunk 1 above)
Comment (Time-Green) [reply code/modules/mob/living/carbon/human/species_types.dm]: Fuck

File: code/modules/mob/living/carbon/human/species_types.dm (hunk 2 above)
Comment (Time-Green) [reply code/modules/mob/living/carbon/human/species_types.dm]: Hmm' I only tested for a few minutes. I'll change the prob to 10.
I am open to alot of suggestions, as I am not good with balancing

**Comment by Time-Green** [comment]
I think it would make sense for them to eat, as plants need to eat a fucking lot.
That's the point of throwing shit at your plants, because it EATS THE NUTRIMENTS

I could just make them eat shit.

EDIT: Eating is fun

**Comment by ShadowDeath6** [comment]
Make water give them nutriment. Food is lame.

**Comment by Time-Green** [comment]
Might not actually be a bad idea shadow.

I'll check back in tomorrow and look at given suggestions

**Comment by ghost** [comment]
what about you know

# my species changes cruix is doing

**Pull request closed without merging**